bindings/tpython/CMakeLists.txt

```diff
@@ -23,7 +23,8 @@ ROOT_STANDARD_LIBRARY_PACKAGE(ROOTTPython
   DEPENDENCIES
     Core
   LIBRARIES
-  cppyy${PYTHON_UNDER_VERSION_STRING_Development_Main}
+    cppyy${PYTHON_UNDER_VERSION_STRING_Development_Main}
+    ${PYTHON_LIBRARIES_Development_Main}
```

Comment:
I would add a comment here, like e.g. : `We link libTPython against Python libraries to compensate for the fact that libcppyy is built with unresolved symbols. If we didn't do this, invoking TPython from C++ would not work.`